roborio-src/constants.py

```diff
@@ -41,3 +42,9 @@
 
 kDriverControllerPort = 0
 kOperatorControllerPort = 1
+
+#Xbox controller constants
+deadband = 0.15
+debouncePeriod = 0.5
+translationSlew = (SlewRateLimiter(1.5), SlewRateLimiter(1.5)) #(Xslew, ySlew)
```

Comment:
translationSlew should be a single value (float), in this case 1.5, that is applied to both the X and Y axis output of the controller.  The slew imposes an acceleration limiter and we want the same limiter to be applied to the X and Y axis.  This keeps the driver from just jamming the joystick all the way to the edge and causing the robot to "jump" to try to match the commanded speed.  We want the same rate for X and Y.  So just one value here, (translationSlew = 1.5)  and we don't want the SlewRateLimiter in the constants.py file.